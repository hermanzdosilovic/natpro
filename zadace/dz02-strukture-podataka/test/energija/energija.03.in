100
ADD 73 24
ADD 30 4
ADD 46 20
ADD 79 8
ADD 25 18
ADD 63 1
DEL 79
QUERY 64
DEL 63
QUERY 244
DEL 25
ADD 74 21
QUERY 375
QUERY 137
ADD 1 32
QUERY 240
QUERY 95
QUERY 113
QUERY 370
ADD 14 16
EXIT
