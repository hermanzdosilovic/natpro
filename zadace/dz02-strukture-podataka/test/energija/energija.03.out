25
25
30
30
1
1
1
1
